util/buildbot/buildwin32.sh

```diff
@@ -113,8 +113,7 @@ cmake -S $sourcedir -B . \
 	-DENABLE_FREETYPE=1 \
 	-DENABLE_LEVELDB=1 \
 	\
-	-DIRRLICHT_INCLUDE_DIR=$libdir/irrlicht/include/irrlichtmt \
-	-DIRRLICHT_LIBRARY=$libdir/irrlicht/lib/libIrrlichtMt.dll.a \
+	-DCMAKE_PREFIX_PATH=$libdir
```

Comment:
you're missing a `\` at the end
and I doubt this works anyway since cmake doesn't recursively search for `.cmake` files

Reply:
find_package searches a predefined list of directories starting from different paths including CMAKE_PREFIX_PATH. As long as the parts aren't in strange locations within the install it works as if it were a system install.

Reply:
The missing \ could explain some probrems, I'll get it fixed quickly.